src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -579,6 +588,12 @@ ParamBlockDesc2 g_param_block_desc(
         p_accessor, &g_pblock_accessor,
     p_end,
 
+	ParamIdRoughnessClamping, L"roughness_clamping", TYPE_BOOL, P_TRANSIENT, 0,
+		p_ui, ParamMapIdLighting, TYPE_SINGLECHEKBOX, IDC_CHECK_ROUGHNESS,
+		p_default, TRUE,
```

Comment:
Default should be FALSE.

Reply:
I have set to default false in the new version